content/docs/16-component.md

```diff
@@ -93,8 +93,18 @@ The page lists the version details of various Palette components with respect to
 # Kubernetes Versions
 
 <Tabs>
+
 <Tabs.TabPane tab="Kubernetes" key="Kubernetes">
 
+## Kubernetes Version Support for Managed Clusters
+
+|Managed Cloud|Version |Reference Release Calender|
+|-------------|--------|--------------------------|
+|Amazon EKS   |1.20.x and above|[Amazon Release Calender](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#:~:text=.-,Amazon%20EKS%20Kubernetes%20release%20calendar,-Note)
+|Azure AKS    |1.21.x and above|[Azure Release Calender](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar)|
```

Comment:
1.24 is GA on in May 2022 per https://kubernetes.io/blog/2022/05/03/kubernetes-1-24-release-announcement/ . So Azure AKS 1.21 is EOL. Please double check